projects/vdk-plugins/vdk-meta-jobs/.plugin-ci.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
> Additionally, this PR includes a necessary fix stemming from a change in the vdk-control-service-api

There are 45 files now added (back). What's the exact change made, to address the api client breaking change?

Reply:
The change is on line 103 in the remote_data_job.py file. Previously, we would assign the return of that method to `_, _, headers`, however now take the headers field from the return object.

Reply:
It's helpful to also link the breaking change itself, to keep track. I didn't realise there were changes with the execution api?